docs/usage-guide/src/error_handling.md

```diff
@@ -0,0 +1,77 @@
+# Error Handling
+
```

Comment:
Should we include a link to here from api.md? Might be out of scope, but I'm a little worried about anyone calling any of our APIs without having read this first.